CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Changes by Version
 ==================
 
+1.26.0 (2021-09-09)
+-------------------
+* Add disconnected annotation to csv ([#1536](https://github.com/jaegertracing/jaeger-operator/pull/1536), [@rubenvp8510](https://github.com/rubenvp8510))
+* Sidecar deployment e2e tests ([#1533](https://github.com/jaegertracing/jaeger-operator/pull/1533), [@mfz85](https://github.com/mfz85))
```

Comment:
this is test feature - not suer if we are documenting test features. We were not doing it before.